doc/contributing/docs/images.rst

```diff
@@ -0,0 +1,84 @@
+Images
+======
+
+Images are useful in explanations of concepts and structures.
+When you introduce a term or describe a structure of multiple interconnected parts
+(such as a cluster), consider illustrating it with a diagram. If you explain how to
+use a GUI, check if a screenshot can make the doc clearer.
+
+Note that illustrations should complement the text, not replace it. Even with an image,
+the text should be enough for readers to understand the topic.
+
+Don't overuse images: they are harder to support than text. Use them only if they bring
+an obvious benefit.
+
+Diagrams
+--------
+
+There is a basic set of diagram elements – blocks, arrows, and other – to use in Tarantool docs.
+It is stored in `this Miro board <https://miro.com/app/board/uXjVPbCIj_g=/>`_. It also provides
+basic rules for creating diagrams.
+
+Size
+~~~~
+
+There are two sizes of diagram elements:
+
+*   **M** – bigger elements for usage in diagrams with small number of elements.
+*   **S** – smaller elements for usage in diagrams with big number of elements.
```

Comment:
```suggestion
*   **M** – bigger elements to use in diagrams with a small number of elements.
*   **S** – smaller elements to use in diagrams with a big number of elements.
```
Let's try to avoid verbal nouns unless they are absolutely necessary.